multisig-wallet/Cargo.toml

```diff
@@ -1,15 +1,13 @@
 [project]
 name = "multisig-wallet"
-version = "0.1.0"
+version = "0.0.0"
```

Comment:
Should this be 0?

Reply:
Yes, we are not releasing new applications. They are meant to be maintained.